fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -70,7 +75,6 @@ public String getFactTableName() {
 
     @Override
     public String toString() {
-        return super.toString() + " factTableName: " + getAvailability().getDataSourceNames() + " alignment: " +
-                getTableAlignment();
+        return super.toString() + " factTableName: " + getAvailability().getDataSourceNames();
```

Comment:
Simplify testing, class scanner checks toString method and blows up with this thing.